src/main/java/org/opensearch/performanceanalyzer/metrics/AllMetrics.java

```diff
@@ -789,16 +789,17 @@ public String toString() {
         }
 
         public static class Constants {
-            public static final String PENDING_TASKS_COUNT_VALUE = "Master_PendingQueueSize";
+            public static final String PENDING_TASKS_COUNT_VALUE =
+                    "ClusterManager_PendingQueueSize";
```

Comment:
nit : can be a single line
```suggestion
            public static final String PENDING_TASKS_COUNT_VALUE = "ClusterManager_PendingQueueSize";
```